src/storm/storage/prism/Program.cpp

```diff
@@ -1174,6 +1174,47 @@ Program Program::labelUnlabelledCommands(std::map<uint64_t, std::string> const&
                    this->getObservationLabels(), this->getOptionalInitialConstruct(), this->getOptionalSystemCompositionConstruct(), prismCompatibility);
 }
 
+Program Program::replaceVariableInitializationByInitExpression() const {
+    std::vector<BooleanVariable> newBooleanVariables;
```

Comment:
Is this supposed to be an empty constructor? Then the subsequent lines do not have an effect.